app/helpers/custom_form_error_helper.rb

```diff
@@ -0,0 +1,14 @@
+module CustomFormErrorHelper
+  def errors(object, attribute)
+    error_messages = object.try(:errors).try(:full_messages_for, attribute)
+
+    if error_messages.is_a?(Array)
+      error_contents = create_error_div(error_messages)
+    end
+    error_contents || ""
```

Comment:
__[RuboCop]__ Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. ([link](https://sider.review/gh/repos/47168038/pulls/585#issue-4232494))

<sub>posted by Sider</sub>